File: tests/typesafety/python/any_arguments.py
# expect: passes
# title: With Any, the checker accepts the arguments in any order
from typing import Any


def invoice_line(description: Any, price_pence: Any, quantity: Any, taxed: Any) -> Any:
    note = " +tax" if taxed else ""
    return f"{description} x{quantity} at {price_pence}d{note}"


line = invoice_line(80, "handlebar grip", True, 2)

File: tests/typesafety/typescript/any_arguments.ts
// expect: passes

function orderLine(name: any, unitPrice: any, quantity: any, gift: any): any {
  const note = gift ? " (gift)" : "";
  return `${name} x${quantity} at ${unitPrice.toFixed(2)}${note}`;
}

export const line = orderLine(3, "tea", true, 1.95); // accepted, and fails at run time
